apps/guide/src/content/03-topics/05-intents.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: Intents
+category: Topics
+---
+
+# Intents
+
+Intents are an important part of establishing a WebSocket connection, as they define behavior regarding gateway events and impact received data via the REST API.
+
+## Usage
+
+```js
+import { Client, GatewayIntentBits } from 'discord.js';
+
+const client = new Client({
+	intents: [GatewayIntentBits.Guilds],
+});
+```
+
+This is the most basic usage of intents for discord.js. By specifying _`GatewayIntentBits.Guilds`_, your bot will receive gateway events regarding guilds. This includes receiving initial information about guilds it is in at startup, such as role data and the shard id to be used.
```

Comment:
sharding has nothing to do with intents! its a separate thing, unrelated to intents. Guild#shardId is a custom prop by us 👀

Reply:
Yes, that's the point. This is a discord.js guide, not a Discord guide.

Reply:
`and the shard id to be used` is incorrect tho, this intent (or any for that matter) have nothing to do with this

Reply:
Resolved with 93eed36d2d2e0c47fece9c6064736ef875e61f6c.